new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,42 @@
+import {downloadBlob} from "../utils/file";
+
+
+class LogExportManager {
+    readonly #chunks: string[];
```

Comment:
It's a preference thing so feel free to object - if the property initialization has nothing to do with the constructor arguments, you can do initialization assignment together with the declaration.

```suggestion
    readonly #chunks: string[] = [];
```
and remove the assignment in the constructor.